.github/workflows/create-release-on-branch-changes.yml

```diff
@@ -6,6 +6,12 @@ on:
       # Dependabot does not have permissions to create releases
       - 'dependabot/**'
 
+# Detect if this action is already running, and cancel it.
+# This most likely happened because a second push has been made to a branch.
+concurrency:
+  group: ${{ github.repository_id }}-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
```

Comment:
I wonder if this is wise.

I fear we end up in situations where a job is half done e.g. created as GitHub release but not as an NPM package.